packages/create-discord-bot/src/helpers/packageManager.ts

```diff
@@ -47,19 +47,52 @@ export function resolvePackageManager(): PackageManager {
  * @param packageManager - The package manager to use
  */
 export function install(packageManager: PackageManager) {
-	let installCommand;
+	let installCommand: string[] | string = `${packageManager} install`;
+
+	console.log(`Installing dependencies with ${packageManager}...`);
 
 	switch (packageManager) {
```

Comment:
just change it to 
if (packageManager === 'yarn')

and use one console.log

Reply:
I guess thats true.

Reply:
But I also want to use the console log only on yarn, pnpm, and deno. I can make a default case and get rid of one statement and make a default instead.